Formula/boost-build.rb

```diff
@@ -2,7 +2,7 @@ class BoostBuild < Formula
   desc "C++ build system"
   homepage "https://www.boost.org/build/"
   url "https://github.com/boostorg/build/archive/boost-1.70.0.tar.gz"
```

Comment:
I think we lost the url change here

Reply:
Wow, that's odd. Not sure how I managed that.